### ISSUE TITLE
No push messages after updating Domoticz

### ISSUE THREAD

**Issue by Olifant1990** [domoticz/domoticz#6114]
Since I updated to the latest beta version (2024.5 (build 16133)) on past sunday I cannot receive Push messages on my android anymore.

I think it has something to do with the FCM migration?
On the mobile devices page I see the last seen time of my device is being updated when I open the app.
The green test arrow to test push is not doing anything.

Any more reports on this?

Maybe @kiddigital can spread some light on this?

Maybe I can find some time to debug this later this week, but the issue is reported for now :)

**Comment by gizmocuz** [comment]
Maybe you have to ask on the Forum fist?

**Comment by Olifant1990** [comment]
No?
It is a code bug somewhere since update to latest FCM.
I updated Domoticz past sunday, opened the android app to update a device code eventually.
This morning I missed some push notifications I send by default with a script.
Then I opened the mobile device page and pushed the test button to receive a push notification,
That doesn't work.

How should I discuss this on the forum?
Somewhere there is a bug.
And it is not configuration.
